src/pages/organize/[orgId]/projects/[campId]/areaassignments/[areaAssId]/report.tsx

```diff
@@ -94,216 +111,352 @@ const AreaAssignmentReportPage: PageWithLayout<AreaAssignmentReportProps> = ({
-                  </Box>
+                            {metric.question || 'Untitled question'}
+                            <Button
+                              onClick={() => {
+                                if (metricBeingDeleted) {
+                                  const filtered = assignment.metrics.filter(
+                                    (metric) =>
+                                      metric.id != metricBeingDeleted.id
+                                  );
+                                  updateAreaAssignment({
+                                    metrics: [
+                                      ...filtered.slice(
+                                        0,
+                                        filtered.indexOf(metric)
+                                      ),
+                                      {
+                                        ...metric,
+                                        definesDone: true,
+                                      },
+                                      ...filtered.slice(
+                                        filtered.indexOf(metric) + 1
+                                      ),
+                                    ],
+                                  });
+                                }
+                                setAnchorEl(null);
+                                setMetricBeingDeleted(null);
+                              }}
+                              variant="outlined"
+                            >
+                              select
+                            </Button>
+                          </Box>
+                        ))}
+                    </Box>
+                  ) : (
+                    <Box display="flex" justifyContent="end" p={2}>
+                      <Button
+                        onClick={() => {
+                          setMetricBeingDeleted(null), setAnchorEl(null);
+                        }}
+                        sx={{ marginRight: 2 }}
+                      >
+                        CANCEL
+                      </Button>
+                      <Button
+                        onClick={() => {
+                          if (metricBeingDeleted !== null) {
+                            handleDeleteMetric(metricBeingDeleted.id);
+                            setAnchorEl(null);
+                            setMetricBeingDeleted(null);
+                          }
+                        }}
+                        variant="contained"
+                      >
+                        CONFIRM
+                      </Button>
+                    </Box>
+                  )}
                 </Box>
               </Dialog>
-            </>
-          )}
-        </ZUIFuture>
-      </Box>
-    </>
+            </Box>
+          </Box>
+          <Box ml={2} width="40%">
+            <ZUICard
+              header="Successful visit"
+              subheader=" Question to use to count a visit as successful."
```

Comment:
This text needs to be internationalized, but I also recommend changing it to something more active, like "Pick a question to use when counting successful visits. Answering yes to this question will count the visit as successful".